prometheus_aioexporter/script.py

```diff
@@ -164,6 +181,21 @@ def _configure_registry(self, include_process_stats: bool = False) -> None:
                 ProcessCollector(registry=None)
             )
 
+    def _get_ssl_context(self, args: argparse.Namespace) -> SSLContext | None:
+        if args.ssl_private_key is None or args.ssl_public_key is None:
+            return None
+        cafile = None
+        if args.ssl_ca:
+            cafile = args.ssl_ca.name
+        ssl_context = ssl.create_default_context(
+            purpose=ssl.Purpose.CLIENT_AUTH, cafile=cafile
+        )
+        ssl_context.load_cert_chain(
+            args.ssl_public_key.name, args.ssl_private_key.name
```

Comment:
since  `FileType` returns an open file, it'd be good to `close()` them here

Reply:
nice one! I am closing all of them in place.